worklenz-backend/Dockerfile

```diff
@@ -1,15 +1,21 @@
-# Use the official Node.js 18 image as a base
-FROM node:18
+# FIRST STAGE: Install Dependencies
+
+# Use the official Node.js 18-alpine3.20 image as a base
+FROM node:18 as builder
```

Comment:
Why not use the same `FROM node:18-alpine3.20 as build`? Also, the naming convention is not consistent, in one image the first stage is named build and another is builder. See `worklenz-frontend/Dockerfile:2` of your PR.